lib/inputs/legacy-dovetail-downloads.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const timestamp = require('../timestamp');
+const lookip = require('../lookip');
+const lookagent = require('../lookagent');
+const bigquery = require('../bigquery');
+
+/**
+ * Send dovetail downloads to bigquery
+ */
+module.exports = class LegacyDovetailDownloads {
+
+  constructor(records) {
+    this._records = (records || []).filter(r => this.check(r));
+  }
+
+  check(record) {
+    return record.type === 'download';
```

Comment:
Using this we we'll be able to support the rollover between the new combined type and existing types ^^